notebooks/Neighborhood applied with kmeans.ipynb

```diff
@@ -0,0 +1,1399 @@
+{
```

Comment:
<p>The Neighborhood category names are scrambled on x axis. Although it's not the most important thing of the notebook, it would be great to fix it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>